Java/linting-rules.xml

```diff
@@ -30,7 +30,7 @@
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
         <module name="LineLength">
-            <property name="max" value="180"/>
```

Comment:
Is there a way to format the code with line breaks to pass this validation instead of changing it?

Reply:
The line exceeds the limit by one character. The possibles solutions are
- Split the method parameters 
- Remove an indentation tab to keep the same format of the other method calls

Reply:
![image](https://user-images.githubusercontent.com/37461749/51708308-5c2d3200-2002-11e9-9ec5-6c470a5f5f1c.png)

Reply:
Which file is this one?

Reply:
I think indentation is wrong right now. I used Reformat Code in IntelliJ and it reduced the indentation by one tab (4 spaces). if that fixes the issue, go ahead. If not, add the method argument splitting.
![image](https://user-images.githubusercontent.com/805847/51708786-8af7d800-2003-11e9-889b-26f4a4d8362e.png)

Reply:
BaseDateTimePeriodParser.java 
Line 94 is breaking the rule.